PySDM/physics/constants_defaults.py

```diff
@@ -63,6 +63,22 @@
 FWC_I7 = 0.105785160e-11 * si.hPa / si.K**7
 FWC_I8 = 0.161444444e-14 * si.hPa / si.K**8
 
+L77W_A0 = 6.107799961 * si.hPa
+L77W_A1 = 4.436518521e-1 * si.hPa / si.K
+L77W_A2 = 1.428945805e-2 * si.hPa / si.K**2
+L77W_A3 = 2.650648471e-4 * si.hPa / si.K**3
+L77W_A4 = 3.031240396e-6 * si.hPa / si.K**4
+L77W_A5 = 2.034080948e-8 * si.hPa / si.K**5
+L77W_A6 = 6.136820929e-11 * si.hPa / si.K**6
+
+L77I_A0 = 6.109177956 * si.hPa
+L77I_A1 = 5.03469897e-1 * si.hPa / si.K
+L77I_A2 = 1.886013408-2 * si.hPa / si.K**2
```

Comment:
The tests are failing because of the units here? I don't really understand the error message, @slayoo.

Reply:
see https://github.com/atmos-cloud-sim-uj/PySDM/pull/759/commits/21ba41d69e687cffa327aa2814152dda9ecf5912 :)

Reply:
Ah! Good eye!